### PR TITLE
Bug-11614: Improve Logo Format

### PR DESCRIPTION
**Pull request by JonathanAsbury-ACS**
Addresses issue logo sizing and positioning issues in the following locations:
AB#11614

1. Next to the Application Title ("Tenant Name")
2. Before uploading new logo
3. After uploading a new logo

The following two files located in the webui section have been updated to accommodate the change:

- .shell.scss ->  src/components/shell/shell.scss
- applicationSettings.scss ->  src/components/shell/flyouts/settings/applicationSettings.scss
 
The rest of the updates are eslint & prettier issues with the --fix generating significant file updates.  This is likely because initial repo was created from a macOS?  Not sure, though.

### CODE REVIEW

File: src/webui/src/components/shell/flyouts/settings/applicationSettings.scss [hunk 1]
```diff
@@ -6,13 +6,12 @@
...
       .image-preview {
-        @include square-px-rem(24px);
         @include rem-fallback(margin-bottom, 8px);
 
         .logo-svg,
         .logo-img  {
-          justify-content: flex-start;
-          @include square-px-rem(24px);
+          max-height: 24px;
+          vertical-align: middle;
```
Comment (JonathanAsbury-ACS) [inline src/webui/src/components/shell/flyouts/settings/applicationSettings.scss:14]: Removes dynamically generate height and with settings that are fixed and cause image resigning, this update sets only the alignment and a fixed height, with will scale automatically and be proportional to the original size.

File: src/webui/src/components/shell/flyouts/settings/applicationSettings.scss [hunk 2]
```diff
@@ -94,7 +93,8 @@
 
     .current-logo {
       @include rem-fallback(margin-right, 12px);
-      @include square-px-rem(24px);
+      max-height: 24px;
+      //@include square-px-rem(24px);
```
Comment (JonathanAsbury-ACS) [inline src/webui/src/components/shell/flyouts/settings/applicationSettings.scss:97]: Same as above but on the upload form .  This style is for the currently uploaded image.

File: src/webui/src/components/shell/flyouts/settings/applicationSettings.scss [hunk 3]
```diff
@@ -104,7 +104,11 @@
     }
   }
 
-  .logo-svg svg { @include square-px-rem(24px); }
+  .logo-svg svg { 
+    //@include square-px-rem(24px); 
+    max-height: 24px;
+    vertical-align: middle;
+  }
```
Comment (JonathanAsbury-ACS) [inline src/webui/src/components/shell/flyouts/settings/applicationSettings.scss:111]: Same as above but on the upload form .  This style is for the newly uploaded image.

File: src/webui/src/components/shell/shell.scss [hunk 4]
```diff
@@ -22,18 +22,13 @@ $companySvgSize: 23px;
...
+      vertical-align: middle;
       @include rem-fallback(margin, 10px);
     }
   }
 
-  .nav-item-icon img { @include square-px-rem($companySvgSize); }
-
-  .nav-item-icon svg { @include square-px-rem($companySvgSize); }
-
   .nav-item-text { text-transform: capitalize; }
```
Comment (JonathanAsbury-ACS) [inline src/webui/src/components/shell/shell.scss:32]: Same as above but for the application title logo (next to the current Tenant Name).

File: src/webui/src/components/shell/flyouts/settings/applicationSettings.scss (hunk 3 above)
Comment (JonathanAsbury-ACS) [reply src/webui/src/components/shell/flyouts/settings/applicationSettings.scss:111]: why is this forcing a resolve.  I didn't request a response.

**Comment by JonathanAsbury-ACS** [comment]
> looks like src/webui/package.json and src/webui/src/app.config.js are both removed in this PR, is that intentional?

Yes, it's intention.  They weren't meant to be checked in.  There's a lot of jumping around for us across computers and accounts for us to use the new Repo, Branches, and PRs so I deleted them from this interface.

**Comment by joe-bethke** [comment]
I'm under the impression that `src/webui/package.json` was necessary in source control, it contains the dependencies for the project. [npm docs](https://docs.npmjs.com/files/package.json)

I also feel like `src/webui/src/app.config.js` is also necessary for building the webui. I don't think these files should be removed or ignored. Your PR removes those files from the repo, if you want to instead just get rid of your changes to those files, I think you can rebase on master, or just commit changes that erase the changes you made and reverts them back to master.

**Pull request closed without merging**